keps/sig-storage/3107-csi-nodeexpandsecret/README.md

```diff
@@ -220,53 +237,125 @@ N/A
 
 ### Rollout, Upgrade and Rollback Planning
 
-TBD
-
 ###### How can a rollout or rollback fail? Can it impact already running workloads?
 
-TBD
+<!--
+Try to be as paranoid as possible - e.g., what if some components will restart
+mid-rollout?
 
-###### What specific metrics should inform a rollback?
+Be sure to consider highly-available clusters, where, for example,
+feature flags will be enabled on some API servers and not others during the
+rollout. Similarly, consider large clusters and how enablement/disablement
+will rollout across nodes.
+-->
 
-TBD
+###### What specific metrics should inform a rollback?
 
+<!--
+What signals should users be paying attention to when the feature is young
+that might indicate a serious problem?
+-->
 ###### Were upgrade and rollback tested? Was the upgrade->downgrade->upgrade path tested?
 
-TBD
-
+<!--
+Describe manual testing that was done and the outcomes.
+Longer term, we may want to require automated upgrade/rollback tests, but we
+are missing a bunch of machinery and tooling and can't do that now.
+-->
 ###### Is the rollout accompanied by any deprecations and/or removals of features, APIs, fields of API types, flags, etc.?
 
-TBD
-
+<!--
+Even if applying deprecation policies, they may still surprise some users.
+-->
 ### Monitoring Requirements
```

Comment:
Questions in this section need to be answered.

Reply:
Done.